disks-and-volumes.md

```diff
@@ -21,6 +21,10 @@ A disk can be made accessible via four different types:
 All possible configuration options are available in the
 [Disk API Reference](https://kubevirt.github.io/api-reference/master/definitions.html#_v1_disk).
 
+All types but **floppy** allow you to specify the `Bus` attribute. The `Bus` attribute determine
```

Comment:
could you use `bus` instead of `Bus`? The user works with json/yaml and not with the struct names.

Reply:
s/determine/determines/